en/assembly/quick_start_pixhawk4.md

```diff
@@ -1,22 +1,26 @@
 # Pixhawk 4 Wiring Quick Start
 
-This quick start guide shows how to power the Pixhawk<sup>&reg;</sup> 4 flight controller and connect its most important peripherals.
-  
-![Pixhawk 4 Image](../../images/pixhawk4_logo_view.png) 
+This quick start guide shows how to power the [Pixhawk 4](../flight_controller/pixhawk4.md)<sup>&reg;</sup> flight controller and connect its most important peripherals.
+
+<img src="../../assets/flight_controller/pixhawk4/pixhawk4_logo_view.jpg" width="420px" title="Pixhawk4 Image" /> 
+
 
 ## Wiring Chart Overview
 
-<p style="margin-bottom:1cm;">The image below shows standard Pixhawk 4 connections (except the motor and servo outputs). We'll go through each main part in the following sections.</p>
+The image below shows how to connect the most important sensors and peripherals (except the motor and servo outputs). We'll go through each of these in detail in the following sections.
 
-![Pixhawk 4 Wiring Overview](../../images/pixhawk4_wiring_overview.png)
+![Pixhawk 4 Wiring Overview](../../assets/flight_controller/pixhawk4/pixhawk4_wiring_overview.png)

```

Comment:
Image is cool, but some of the text might be improved:

Text for Power Management Board near bottom:

> Connect to the Power Management Board using the provided 10 wire cable. This sends PWM signals to the motors.

Text for near top:

> Connect to the Power Management Board using the provided 6 wire cable. This provides power to your Autopilot (from the LiPo battery).